app/hooks/useSettingsConfig.ts

```diff
@@ -162,6 +164,15 @@ const useSettingsConfig = () => {
         group: t("Workspace"),
         icon: ShapesIcon,
       },
+      {
+        name: t("Custom Emojis"),
```

Comment:
```suggestion
        name: t("Emojis"),
```

I'm ok with just "Emojis"